plugin/evm/vm.go

```diff
@@ -636,7 +636,7 @@ func (vm *VM) initializeMetrics() error {
 		return nil
 	}
 
-	gatherer := corethPrometheus.Gatherer(metrics.DefaultRegistry)
+	gatherer := corethprometheus.NewGatherer(metrics.DefaultRegistry)
```

Comment:
nit I took the liberty (again) to rename `Gatherer` to `NewGatherer` in order to export `gatherer` as `Gatherer` and return a `*Gatherer` from `NewGatherer` to satisfy the "return concrete types" concept